packages/autocomplete-js/src/types/AutocompleteRenderer.ts

```diff
@@ -37,4 +37,4 @@ export type AutocompleteRenderer = {
   Fragment: PragmaFrag;
 };
 
-export type HTMLToJSX = typeof htm;
+export type HTMLTemplate = typeof htm;
```

Comment:
Better naming indeed, however this isn't really a "template" compared to the existing `Template` type in Autocomplete, it's more of a template transformer or such. So maybe `HTMLTemplateTransformer`?

Reply:
I'm not sure the distinction is required. It's a "tagged template" which allows the user to write html, so HTMLTemplate describes that enough IMO. Similarly how we call it a widget, and not a widget factory